addons/stock/__openerp__.py

```diff
@@ -32,7 +32,7 @@
 * Moves Analysis
     """,
     'website': 'https://www.odoo.com/page/warehouse',
-    'depends': ['product', 'procurement', 'barcodes', 'web_planner'],
+    'depends': ['product', 'procurement', 'barcodes', 'web_planner', 'l10n_generic_coa'],
```

Comment:
Ohhh ***Odoo*** permits this type of module into depends :question:  @tde-banana-odoo check once , Thank you :+1: